boards/nucleo-f303ze/Kconfig

```diff
@@ -21,6 +21,8 @@ config BOARD_NUCLEO_F303ZE
     select HAS_PERIPH_SPI
     select HAS_PERIPH_TIMER
     select HAS_PERIPH_UART
+    select HAS_PERIPH_USBDEV
+    select HAS_TINYUSB_DEVICE
```

Comment:
Now this has to be removed after all 

```suggestion
```

Reply:
Oops, I shouldn't push changes while sitting in a meeting :worried:

Reply:
That depends entirely on the meeting :smile: